src/main/java/org/sunbird/cb/hubservices/profile/handler/ProfileRequestHandler.java

```diff
@@ -48,10 +47,12 @@ public RegistryRequest updateRequest(String uuid, Map<String, Object> request) {
 		// search with user id
 		ResponseEntity<?> responseEntity = profileUtils.getResponseEntity(ProfileUtils.URL.SEARCH.getValue(),
 				searchRequest(uuid));
-
-		Object searchResult = ((Map<String, Object>) ((Map<String, Object>) responseEntity.getBody()).get("result"))
-				.get(ProfileUtils.Profile.USER_PROFILE);
-
+		Object searchResult ="";
+		Object responseBody =responseEntity.getBody();
+		if(null!=responseBody) {
+			 searchResult = ((Map<String, Object>) ((Map<String, Object>) responseBody).get("result"))
+					.get(ProfileUtils.Profile.USER_PROFILE);
+		}
```

Comment:
This part of code is not written properly. 
what happens when responseEntity.getBody() is null ??